R/distro.R

```diff
@@ -25,9 +25,9 @@ distro <- function() {
   }
 
   out$id <- tolower(out$id)
-  if (grepl("bullseye", out$codename)) {
+  if (grepl("bookworm", out$codename)) {
```

Comment:
I wonder if we should support both cases here (since not everyone has updated their debian:unstable to bookworm yet), something like (you'll have to fix the suggestion on the other side to get the close brace and indentation, GitHub won't let me suggest across the whole diff)

```suggestion
  if (is.null(out$version)) {
    if (grepl("bullseye", out$codename)) {
      out$short_version <- "11"
    else if (grepl("bookworm", out$codename)) {
```